sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -227,9 +227,12 @@ def test_predict_time_index(
 
         # if estimator cannot forecast in-sample and fh is in-sample, terminate
         # if the tag correctly states this, we consider this fine as per contract
-        # todo: check that estimator raises error message when fitting instead
+        # check that estimator raises error message when fitting
```

Comment:
use `with pytest.raises` instead, that is the idiomatic check for an exception being raised. Also check that the error message agrees.

Reply:
done